app/src/main/java/com/pvp/app/ui/screen/goals/GoalScreen.kt

```diff
@@ -108,6 +110,28 @@ fun GoalScreen(
                                 )
                             }
                         }
+
+                        item {
+                            Spacer(modifier = Modifier.padding(30.dp))
+
+                            Text(
+                                style = MaterialTheme.typography.titleLarge,
+                                modifier = Modifier.fillMaxWidth(),
+                                textAlign = TextAlign.Center,
+                                text = "Completed:"
+                            )
```

Comment:
If we drop `:` and align text to the left, I think it will look better, just like with settings screen.
Possibly less space around the title will be needed too, try to make it similar to settings screen.